Oqtane.Server/Startup.cs

```diff
@@ -180,43 +114,13 @@ public void ConfigureServices(IServiceCollection services)
             // register custom claims principal factory for role claims
             services.AddTransient<IUserClaimsPrincipalFactory<IdentityUser>, ClaimsPrincipalFactory<IdentityUser>>();
 
-            // register singleton scoped core services
             services.AddSingleton(Configuration);
```

Comment:
I don't think this line is necessary, but the reason for that is you are used `IConfigurationRoot` in `DatabaseMigration`